content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,120 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a
+[valid URL format](#url-format-to-specify-folder-location). Then use any DVC
+command that needs it (e.g. `dvc pull`, `dvc fetch`, `dvc push`). For example:
+
+```dvc
+$ dvc remote add -d gdremote gdrive://0AIac4JZqHhKmUk9PDA/dvcstore
+$ dvc push
+...
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/auth # ... copy this link
+
+Enter verification code: # <- enter resulting code
+```
+
+See the [Authorization](#authorization) section for more details on connecting
+DVC with Google Drive.
+
+DVC uses the Google Drive API to synchronize your <abbr>DVC project</abbr> data
+with this type of remote storage, so it's subject to certain usage limits and
+quotas, which by default are shared with other GDrive remote storage users. For
+heavy use, it's highly recommended to
+[connect using a custom Google Cloud project](#connect-using-a-custom-google-cloud-project),
+which puts you in control of these limits.
+
+Having your own GC project, it's also possible to
+[use a service account](#using-service-accounts) for automating tasks that need
+to establish GDrive remote connections (e.g. CI/CD).
+
+## URL formats to specify folder location
+
+There's a few alternatives to construct a GDrive remote URL for different uses,
+such as a folder or subfolder in root, shared folders not owned by your account,
+etc. The URL is formed with a _base_, and an optional _path_ to an **existing**
+folder i.e. `gdrive://<base>/path/to/folder`. The base can be one of:
+
+1. _Folder ID_ - unique identifier for each Google Drive folder
+
+   To obtain it, navigate to that folder in your web browser, and find it in the
+   address bar. For example, for
+   `https://drive.google.com/drive/folders/0AIac4JZqHhKmUk9PDA`:
+
+   ```dvc
+   $ dvc remote add gdfolder gdrive://0AIac4JZqHhKmUk9PDA
+   $ dvc remote add gdsubfolder \
+                  gdrive://0AIac4JZqHhKmUk9PDA/Data/text
+   ```
+
+   Note that [shared folders](https://support.google.com/drive/answer/7166529)
+   and [shared drives](https://support.google.com/a/users/answer/9310351) can
+   only be referenced this way.
+
+2. `root` - indicates your topmost Google Drive directory.
+
+   ⚠️ Only suitable for personal use, as sharing a remote configured this way
+   would cause DVC to try synchronizing data to/from different Google Drives for
+   every user.
+
+   ```dvc
+   $ dvc remote add mygdfolder gdrive://root/dvcstore
+   ```
+
+   > Although valid, we don't recommend using just `gdrive://root`, as it's not
```

Comment:
it will be very hard to understand I believe

Reply:
Changed to
   > We don't recommend using `gdrive://root` itself, as it's likely used for
   > many other reasons, and pushing data with DVC here can make it seem messy.

WDYT?

Reply:
👍